schema/acq-operation.json

```diff
@@ -1,10 +1,71 @@
 {
-  "id": "acq-.json",
+  "id": "acq-operation.json",
   "$schema": "https://json-schema.org/draft-04/schema#",
   "description": "",
   "type": "object",
-  "example": "../examples/acq-.json",
+  "example": "../examples/acq-operation.json",
   "additionalProperties": false,
-  "required": [],
-  "properties": {}
+  "required": ["operationId","identifier","status","parentOperation"],
+  "properties": {
+    "operationId": {
+      "type": "string",
+      "description": "a unique identifier for the operation"
+    },
+    "description": {
+      "type": "string",
+      "description": "description of the mission on which the platform observations are made and the objectives of that mission"
+    },
+    "citation": { 
+      "$ref": "./citation.json",
+      "description": "identification of the mission"
+    },
+    "identifier": { 
+      "$ref": "./identifier.json",
+      "description": "unique identification of the operation"
+    },
+    "status": {
+      "type": "string",
+      "description":  "status of the data acquisition"
```

Comment:
Needs enum with the allowed values (this list comes from the MD_ProgressCode code list)